modules/management-console-agent/src/main/java/org/apache/ignite/agent/action/controller/QueryActionsController.java

```diff
@@ -132,9 +133,13 @@ public QueryActionsController(GridKernalContext ctx) {
 
                 SqlFieldsQuery qry = prepareQuery(arg);
 
-                GridCacheContext cctx = F.isEmpty(arg.getCacheName())
+                IgniteInternalCache<Object, Object> cache = F.isEmpty(arg.getDefaultSchema())
+                    ? null
+                    : ctx.cache().cache(arg.getDefaultSchema());
```

Comment:
org.apache.ignite.internal.processors.cache.GridCacheProcessor#cache use the cache name as an argument, but you pass schema name. How this will work?

Reply:
Expected that cache name will be sent and its context will be used.